Mesh4all/m4a-firmware
# peripherals/doc; Added Documentation to Peripherals

### Contribution description
In this PR was added information to DS18b20 and Analog Capacitive Moisture Sensor 1.2V, gives more information about the kernel configurations previously to the compilation of peripherals. And add an general introduction to peripherals documentation.



### Testing procedure
type `make docs` then open the `index.html` from doxygen/html folder and see the section peripherals



### Issues/PRs references
Fixes #272 
Fixes #277 
Fixes #278
